api2-war/src/main/java/eu/europeana/api2/v2/utils/ControllerUtils.java

```diff
@@ -152,4 +152,16 @@ private static String removeProfileFromRequest(String query) {
         return query.replaceAll(PROFILE_PATTERN, "");
     }
 
+
+    /**
+     * Return true is status code is 5xx
+     * @param httpStatusCode
+     * @return
+     */
+    public static boolean is5xxError(int httpStatusCode) {
+        return (httpStatusCode == 500 || httpStatusCode == 501 ||
+                httpStatusCode == 502 || httpStatusCode == 503 ||
+                httpStatusCode == 504 || httpStatusCode == 505 ||
+                httpStatusCode == 507) ;
```

Comment:
I'm not sure what the implementation is on the Translation API side, but in theory other 5xx error could be returned. It safer to check if the number is >= 500 and <= 599

Reply:
have updated this implmenation